test/screenshot_script.sh

```diff
@@ -5,29 +5,65 @@ set -e
 # dir of bash script http://stackoverflow.com/questions/59895
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-#read data1 - normal data set
+#read data1 - normal data set (months)
 data1file=${DIR}'/data/data1.json'
 data1=$(cat "$data1file")
 
-#read data2 - for beginning at 0
+#read data2 - for beginning at 0 (days)
 data2file=${DIR}'/data/data2.json'
 data2=$(cat "$data2file")
 
+#read data3 (years)
+data3file=${DIR}'/data/data3.json'
+data3=$(cat "$data3file")
+
 # patient view screenshot
 phantomjs --ignore-ssl-errors=true ${DIR}/make_screenshots.js \
 	${DIR}'/../index.html?json='"$data1" \
-	${DIR}'/screenshots/index_html_data1.png' \
+	${DIR}'/screenshots/index_html_data1_advanced.png' \
 	50
 
 phantomjs --ignore-ssl-errors=true ${DIR}/make_screenshots.js \
 	${DIR}'/../index.html?json='"$data2" \
-	${DIR}'/screenshots/index_html_data2.png' \
+	${DIR}'/screenshots/index_html_data2_advanced.png' \
+	50
+
+phantomjs --ignore-ssl-errors=true ${DIR}/make_screenshots.js \
+	${DIR}'/../index.html?json='"$data3" \
+	${DIR}'/screenshots/index_html_data3_advanced.png' \
+	50
+
+phantomjs --ignore-ssl-errors=true ${DIR}/make_screenshots.js \
+	${DIR}'/../index.html?json='"$data1"'?view=simple' \
+	${DIR}'/screenshots/index_html_data1_simple.png' \
+	50
+
+phantomjs --ignore-ssl-errors=true ${DIR}/make_screenshots.js \
+	${DIR}'/../index.html?json='"$data2"'?view=simple' \
+	${DIR}'/screenshots/index_html_data2_simple.png' \
```

Comment:
This one looks like cutting is redundant right? It only cuts away one day? It would look better I think if you cut from 3 to 5 and again have the zigzag in the middle.